static/app/components/sidebar/utils.tsx

```diff
@@ -1,8 +1,13 @@
 import type {OnboardingTaskStatus} from 'sentry/types/onboarding';
+import type {Organization} from 'sentry/types/organization';
 
 export const isDone = (task: OnboardingTaskStatus) =>
   task.status === 'complete' || task.status === 'skipped';
 
 // To be passed as the `source` parameter in router navigation state
 // e.g. {pathname: '/issues/', state: {source: `sidebar`}}
 export const SIDEBAR_NAVIGATION_SOURCE = 'sidebar';
+
+export function hasWhatIsNewRevampFeature(organization: Organization) {
+  return organization.features.includes('what-is-new-revamp');
```

Comment:
two nits here - would prefer we call this `whats-new-revamp` if the flag hasn't been created yet. second, do we really need this util function ? i feel like since it is just a feature flag check we could have it be     `const hasWhatsNewRevamp = organization.features.includes()...;`

Reply:
yes initially I thought we would use it in more different places but it makes sense

Reply:
oh and the feature flag is already created